doc/data-warehouse-what-and-why.md

```diff
@@ -0,0 +1,258 @@
+all our content over time, so we can analyze it to make informed decisions about
+content.
+```
+
+If we look into the Wikipedia definition of a Data Warehouse:
+
+```
+A central repository of integrated data from one or more disparate sources, that
+stores current and historical data in one single place that is used for creating
+analytical reports for workers throughout the enterprise.
+```
+
+And, if we replace:
+
+source|replaced
+-------------------------------|--------------------------
+one or more disparate sources | Feedex, Google Analytics
+workers | content designers
+the enterprise | All GOV.UK organizations
+
+We get:
+
+```text
+A central repository of integrated data from Feedex, Google Analytics, PAAS...,
+that stores current and historical data in one single place that is used for
+creating analytical reports for content designers throughout all GOV.UK
+organizations.
+```
+
+So, it seems that from a definition / formal perspective, it fits in what we
+need to track metrics through time.
+
+## Track content changes over time for Content Items
+
+We not only need to track metrics about how content is performing through time,
+but we also need to monitor the content attributes, so we understand the
+relationship between content and performance changes through time.
+
+Say, for example, that a content item was created and published three years ago.
+To get a good understanding of how the content changes are related to their
+performance, we would need to:
+
+1. Track each change to the Content Item through the three years
+2. Update the quality metrics each time a new version is created
+3. Link the performance metrics to each one of the versions of the Content Item.
+
+Tracking the history of each content item is key to understand the relationship
+between content and performance.
+
+## A new application for GOV.UK?
+
+With this context, we decided to build a new application for GOV.UK, a Data
+Warehouse that will track content and performance changes through time and that
+will enable other applications within GOV.UK to get the most of this
+information.
+
+Before building the application, we first checked for the state of the art of
+all the different products within GOV.UK checking if we could re-use some of
+this information, so we don't reinvent the wheel.
+
+This is what we found out:
+
+1. We don't have a single place to retrieve all metrics for content items. We
+   currently store metrics for organizations (and dashboards) via the
+   performance platform, but the way it is structured and the scope of the
+   product, make it not a good fit for our mission goal.
+
+1. We store the number of page views in [Search][rummager], so when users search
+   in GOV.UK the results are ordered by popularity; the way this information is
+   collected [has been reported as Tech debt][ga-tech-debug-trello] and building
+   this applpcation will enable [Search][rummager] to use the API and pay the
```

Comment:
`to use the API` - to use which API? This is the first mention of an "API", and I assume you are talking about content-performance-manager, but this won't be clear for people unfamiliar with the project.

Reply:
I have added a link to the API documentation. Once the application is renamed to `content-data-api` I guess it will be more clear for developer how the product shares information.